lib/generators/curation_concerns/install_generator.rb

```diff
@@ -104,5 +105,12 @@ def resource_types_config
     def simple_form_bootstrap
       generate 'simple_form:install --bootstrap'
     end
+
+    def simple_form_initializers
+      copy_file 'config/initializers/simple_form.rb',
```

Comment:
They only differ by changing the order of the elements so the `hint` comes before the `input`. This is to make it match the design which was approved in projecthydra/sufia#2003 and because having the `hint` come after the `input` is worse for people using some assistive technologies.